internal/state/api/api.go

```diff
@@ -70,6 +70,8 @@ func (s *Server) Init() {
 	s.router.GET(path.Join(base, "guilds", ":guild", "emojis", ":emoji"), wrapHandler(s.log, s.getGuildEmoji))
 
 	s.router.GET(path.Join(base, "guilds_count"), wrapHandler(s.log, s.getGuildCount))
+
+	s.router.GET(path.Join(base, "guilds", ":guild", "threads"), wrapHandler(s.log, s.getGuildThreads))
```

Comment:
there's quite abit of implementation in the other files, but there's only 1 new api endpoint?